recipes-domu/domu-image-android/files/meta-xt-prod-extra/recipes-extended/android/android.bbappend

```diff
@@ -3,7 +3,7 @@ SRCREV = "${AUTOREV}"
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 
 SRC_URI_append = " \
-    repo://github.com/xen-troops/manifests;protocol=https;branch=master;manifest=prod_ces2018/doma.xml;scmdata=keep \
+    repo://git@gitpct.epam.com/epmd-aepr/android_manifest.git;protocol=ssh;branch=master;manifest=doma.xml;scmdata=keep \
```

Comment:
IMO, commit message should also note that Android's xml moves from xen-troops to gitlab

Reply:
Done